suggested offer of help (#23)
fortran-lang/fpm-registry: README.md

Your new registry submittal should first meet the 
[general package criteria](https://github.com/fortran-lang/fortran-lang.org/blob/master/PACKAGES.md)
required of any package listed at the
[packages listing](https://fortran-lang.org/packages).

Please submit a Pull Request ([PR](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github))
against this repository, adding the new package into the file
[registry.toml](./registry.toml)

> in alphabetical order. Explicitly list each version using the 

Reviewer: ```suggestion
in alphabetical order. It is recommended that you explicitly list each version using the 
```

[`tag`](https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/managing-tags)

> keyword, otherwise the master branch will be used. An example:

Reviewer: ```suggestion
keyword, otherwise the tip of the master branch will be used. An example:
```

in alphabetical order. Explicitly list each
version using the 
in alphabetical order. It is recommended that you explicitly list each version using the 
[`tag`](https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/managing-tags)
keyword, otherwise the master branch will be used. An
example:
keyword, otherwise the tip of the master branch will be used. An example:

```toml
[datetime]
"1.7.0" = {git="https://github.com/wavebitscientific/datetime-fortran", tag="v1.7.0"}
"latest" = {git="https://github.com/wavebitscientific/datetime-fortran"}

> ```

Reviewer 2: This line was probably deleted by accident

Your new registry submittal should first meet the 
[general package criteria](https://github.com/fortran-lang/fortran-lang.org/blob/master/PACKAGES.md)
required of any package listed at the
[packages listing](https://fortran-lang.org/packages).

> Please submit a **PR** or
> [pull request](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)

Reviewer 3: ```suggestion
Please submit a Pull Request ([PR](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github))
```

When a pull request is merged it will then appear at the
[`fpm registry`](https://fortran-lang.org/packages/fpm).

## How to request help

> If you have a great Fortran package you have managed to get onto
> **github** or another **git** repository that you are interested
> in setting up as a registered **fpm** package that adheres to the
> guidelines listed here but terms like **fpm**, **PR**, **toml**, and
> **python** are putting you off from registering your package, please open an
> [issue](https://github.com/fortran-lang/fpm-registry/issues) at the
> **fpm Registry** site and list your Repository and ask for help.  Get as
> far as you can and identify where you got stuck. We will try to help.

Reviewer: ```suggestion
If you have a great Fortran package that you are interested
in registering as an **fpm** package but terms like **fpm**, **PR**, **toml**, and
**python** are putting you off, then please open an
[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
**fpm Registry** site listing your Repository and let us know how we can help. Get as
far as you can and identify where you got stuck. We are always happy to help.
```

This is a *very* long multi-clause sentence; I have removed some redundancy and simplified.

When a pull request is merged it will then appear at the
[`fpm registry`](https://fortran-lang.org/packages/fpm).

> ## Are you kidding?

Reviewer 3: ```suggestion
## How to request help
```

If you have a great Fortran package you have managed to get onto
**github** or another **git** repository that you are interested
in setting up as a registered **fpm** package that adheres to the
guidelines listed here but terms like **fpm**, **PR**, **toml**, and

> **python** are putting you off from registering your package open an

Reviewer 3: ```suggestion
**python** are putting you off from registering your package, please open an
```

[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
**fpm Registry** site and list your Repository and ask for help.  Get as

> far as you can and identify where you got stuck.

Reviewer 3: ```suggestion
far as you can and identify where you got stuck. We will try to help.
```
